### PR TITLE
feature - change add to cart logic

### PR DESCRIPTION
**Pull request by at-tanhle**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Home/HomeViewController.swift [hunk 1]
```diff
@@ -198,10 +200,27 @@ extension HomeViewController: HomeFooterSectionViewDelegate {
...
         let vc = ListMenuViewController()
         vc.viewModel = viewModel.listViewModelWith(type: .CockTail)
         navigationController?.pushViewController(vc, animated: true)
     }
 }
+
+// MARK: - HomeCellDelegate
+extension HomeViewController: HomeCellDelegate {
+    func cell(_ cell: HomeCell, needsPerformAction action: HomeCell.Action) {
+        guard let HomeCellVM = cell.viewModel else { return }
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: HomeCellVM?

File: MyApp/View/Controllers/Home/HomeViewController.swift [hunk 2]
```diff
@@ -198,10 +200,27 @@ extension HomeViewController: HomeFooterSectionViewDelegate {
...
+        HomeCellVM.addToCart()
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: HomeCellVM?

File: MyApp/View/Controllers/Home/HomeViewController.swift [hunk 3]
```diff
@@ -198,10 +200,27 @@ extension HomeViewController: HomeFooterSectionViewDelegate {
...
+    }
+}
+
+// MARK: - HomeFavoriteCellDelegate
+extension HomeViewController: HomeFavoriteCellDelegate {
+    func cell(_ cell: HomeFavoriteCell, needsPerformAction action: HomeFavoriteCell.Action) {
+        guard let HomeFavoriteCellVM = cell.viewModel else { return }
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Home/HomeViewController.swift]: HomeFavoriteCellVM?

**Pull request merged**